1-mandatory/1-writers.js

```diff
@@ -51,7 +51,11 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  writers.map((object) => {
```

Comment:
Check if the variable names are clear and descriptive. Make sure they represent their purpose in the code.

When iterating through writers, decide whether you should use map() or another array method, based on what you want to achieve.